x-pack/test/accessibility/apps/security_solution.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const a11y = getService('a11y');
+  const browser = getService('browser');
+  const { common, detections } = getPageObjects(['common', 'detections']);
+
+  describe('Security Solution', () => {
+    before(async () => {
+      await common.navigateToApp('security');
+      await browser.setWindowSize(1600, 1200);
+    });
+
+    describe('Detections', () => {
+      describe('Create Rule Flow', () => {
+        beforeEach(async () => {
+          await detections.navigateToCreateRule();
+        });
+
+        describe('Custom Query Rule', () => {
+          describe('First Step', () => {
```

Comment:
For readability purposes I think that instead of `First Step`, `Second Step` and so on, it is better to use the name of the step: `Define step`, `About step` and so on.